McEngine/src/App/UIFrameworkTest/UIFrameworkTestMenu.h

```diff
@@ -0,0 +1,34 @@
+/*
+ * UIFrameworkTestMenu.h
+ *
+ *  Created on: May 29, 2017
+ *      Author: Psy
+ */
+
+#ifndef APP_UIFRAMEWORKTEST_UIFRAMEWORKTESTMENU_H_
+#define APP_UIFRAMEWORKTEST_UIFRAMEWORKTESTMENU_H_
+
+#include "cbase.h"
+#include "Engine.h"

```

Comment:
Don't include Engine.h in a header file, you don't need it. Only include it in cpp files.